README.md

```diff
@@ -428,6 +437,284 @@ HTTP/1.1 200 OK
 
 This can be treated as a keep-alive control for some load balancers.
 
+### Subscriptions
+Using subscriptions users are able to consume events from event-types in a "high level" way when 
+Nakadi stores the offsets and manages the rebalancing of consuming clients. The clients using 
+subscriptions can stay really stateless. It is possible to subscribe to multiple event-types in 
+one subscription* so that within one connection it is possible to read events from many event-types.
+
+_(\* This possibility will be enabled soon)_
+
+The typical workflow when using subscriptions is following:
+
+1) create subscription specifying the event-types you want to read;
+
+2) start reading events from your subscription;
+
+3) commit the cursors you get in event batches;
+
+If you closed the connection and after some time started reading again - you get events from the 
+point of your latest commit.
+Also, it is not necessary to commit each batch you get. When you commit the cursor, all events that
+are before this cursor will also be considered committed.
+
+If you need more than one client for your subscription to distribute the load - you can start 
+reading the subscription with multiple clients and Nakadi will balance the load among your clients. 
+The balancing units are partitions, so the number of clients of your subscription can't be higher 
+than the total number of all partitions of the event-types of your subscription. 
+
+_E.g. if you have a subscription for two event-types A and B having 2 and 4 partitions accordingly
+then if you start reading events by a single client - you will get events from all 6 partitions
+by this client. If you connect by a second client - 3 partitions will be transferred from first client
+to a second client and each client will be getting data from 3 partitions. So the maximum possible 
+number of clients for this subscription is 6 (in that case each one will read from one subscription)_
+
+Bellow please find basic examples of Subscription API usage. For more detailed description and advanced
+configuration please take a look at Nakadi [swagger](api/nakadi-event-bus-api.yaml) file.
```

Comment:
@BGehrels Ok, thanks for raising these points. 

I added some description here https://github.com/zalando/nakadi/pull/451/commits/ae861e209b677b71f2d8e8ab3b3340ac40fdbcc1

However I didn't describe here that `different partitions will be multiplexed into one stream` because it's not subscription-specific behavior, we had the same behavior in Low Level API, so I think that should be described in more general place. Also I'm not really agree regarding `non-chronological order`, but maybe I just misunderstood what you mean.